src/lib/db/project-store.ts

```diff
@@ -265,25 +266,21 @@ class ProjectStore implements IProjectStore {
             await this.db(TABLE)
                 .where({ id: data.id })
                 .update(this.fieldToRow(data));
-            if (
-                data.defaultStickiness !== undefined ||
-                data.featureLimit !== undefined
-            ) {
-                if (await this.hasProjectSettings(data.id)) {
-                    await this.db(SETTINGS_TABLE)
-                        .where({ project: data.id })
-                        .update({
-                            default_stickiness: data.defaultStickiness,
-                            feature_limit: data.featureLimit,
-                        });
-                } else {
-                    await this.db(SETTINGS_TABLE).insert({
-                        project: data.id,
+
+            if (await this.hasProjectSettings(data.id)) {
```

Comment:
I think now we need to 
```
data.defaultStickiness !== undefined ||
                data.featureLimit !== undefined
```

Reply:
https://github.com/Unleash/unleash/pull/5317/commits/0475ac3e4ad6b0d67df47c74c9dc2216b9442658